source/js/main.js

```diff
@@ -21,8 +21,10 @@ import bindMozFestGA from "./mozfest-ga.js";
 
 const SHOW_MEMBER_NOTICE = false;
 
+const DEFAULT_LOCALE = `en`;
```

Comment:
(I know little about localization) 

Our default locale used to be `en_US` (e.g., ```<meta property='og:locale' content='en_US'>```). Just wanna make sure we are intentionally changing it to just `en` now

Reply:
We're changing it because it's based on the code used by Wagtail. We're using the short codes in wagtail, which are thankfully legal codes: https://kb.yoast.com/kb/changing-the-og-locale-output/#valid

Reply:
So, some vocabulary:

`en` is the language
`US` is the region
`en_US` is the locale

We've never used a full locale on the Foundation site, since it's not supported by default in Django/Wagtail. What's the concern here?

Reply:
opengraph needs locale, wagtail uses language. So we need to generate a locale, from a language, and there's nothing built in for that.

Reply:
I've updated the PR with a mapping so that we can generate the correct opengraph locale on the Django side of the fence.